src/pods/generate/variables/Variables.ts

```diff
@@ -0,0 +1,14 @@
+// Variables used for configuration templates
+export const TEMPLATE_BASE_URL_URN = 'urn:solid-server:template:variable:baseUrl';
+export const TEMPLATE_ROOT_FILEPATH_URN = 'urn:solid-server:template:variable:rootFilePath';
+export const TEMPLATE_SPARQL_ENDPOINT_URN = 'urn:solid-server:template:variable:sparqlEndpoint';
+export const TEMPLATE_CONFIG_URN = 'urn:solid-server:template:variable:templateConfig';
+export const TEMPLATE_STORE_URN = 'urn:solid-server:template:ResourceStore';
```

Comment:
Should we reuse our namespace creation function here?

Reply:
Yea not a bad idea.